javascript/consumer.js

```diff
@@ -4,7 +4,7 @@ function isConsumer (object) {
   if (object) {
     try {
       return (
-        object.constructor.name === 'Consumer' &&
+        object.constructor.name == 'ActionCableAdapter' &&
```

Comment:
The one thing I'm unsure about is this. We need to assume that the constructor name can be different than `ActionCableAdapter`. Is there a way to go around this @julianrubisch ?

Reply:
Also on line 27 in this file. We should consider returning the adapted consumer directly.